shapely/tests/test_cga.py

```diff
@@ -34,8 +38,9 @@ def test_no_holes(self):
         assert polygon.exterior.is_ccw
 
     def test_holes(self):
-        polygon = Polygon([(0, 0), (0, 1), (1, 0)],
-                          [[(0.5, 0.25), (0.25, 0.5), (0.25, 0.25)]])
+        polygon = Polygon(
```

Comment:
Minor nit: consider surrounding this in `# fmt: off` / `# fmt: on` to prevent unwrapping; here it is helpful for the hole to be on a separate line.

Reply:
ok